tests/robot-tests/tests/public_api/public_api_restricted.robot

```diff
@@ -106,6 +106,43 @@ Verify the contents inside the 'Draft API datasets' table
     user checks table cell contains    2    3    Ready    xpath://table[@data-testid='draft-api-data-sets']
     
 
+Click on 'View Details' link
+    user clicks link in table cell    1    4    View details    xpath://table[@data-testid='draft-api-data-sets']
+    user waits until h3 is visible    Draft version details
+    user checks row data heading in the Draft version details table
+
+User checks row data contents inside the 'Draft API datasets' summary table
+    user checks contents inside the cell value    v1.0                                       xpath://dl[@data-testid="draft-version-summary"]/div/dd[@data-testid='Version-value']/strong
+    user checks contents inside the cell value    Ready                                      xpath:(//div[@data-testid="Status"]//dd[@data-testid="Status-value"]//strong)[2]
+    user checks contents inside the cell value    Financial year 3000-01                     xpath:(//div[@data-testid="Release"]//dd[@data-testid="Release-value"]//a)[1]
+    user checks contents inside the cell value    ${SUBJECT_NAME_1}                          xpath://div[@data-testid="Data set file"]//dd[@data-testid="Data set file-value"]
+    user checks contents inside the cell value    National                                   xpath://div[@data-testid="Geographic levels"]//dd[@data-testid="Geographic levels-value"]
+    user checks contents inside the cell value    2012/13                                    xpath://div[@data-testid="Time periods"]//dd[@data-testid="Time periods-value"]
```

Comment:
Is it possible that some of these selectors could be simplified to just `//dd[@data-testid="Time periods-value"]`?

Same for the selectors below

Reply:
I'd be surprised if the selector itself is causing an issue here, but equally I don't feel strongly about this so happy for you to leave it. Assuming there is no issue with the simplified selectors, there is an argument that the time to simplify these selectors for readability isn't worth it anyway.